startServerMac.sh

```diff
@@ -74,6 +74,7 @@ if [ "$freshSync" = true ]; then
 fi
 
 echo "Starting the server now"
+virtualenv venv
```

Comment:
this should not be here
we don't wanna create a new virtual env everytime someone runs this script.
we're assuming that the developer has done that already when they setup this project